README.md

```diff
@@ -18,6 +18,59 @@ Read ["Installing Rust"] from [The Book].
 
 ## Building from Source
 
+Starting from `e645ed59` the Rust Build System is being converted to use
+Cargo. The new build system is supposed to replace the old Makefile-based
+build system once it is mature enough. It is encouraged to use the new build
+system whenever possible as the Makefile-based system is marked for deletion
+in the future.
+
+To build the compiler,
+
+1. Make sure you have installed the dependencies:
+
+   * `g++` 4.7 or `clang++` 3.x
+   * `python` 2.7 or later (but not 3.x)
```

Comment:
3.x is released so long ago, that it doesn’t make sense to support 2.x only. That’s something that can be fixed later, of course.